rest/admin_api_auth_routing_permissions.go

```diff
@@ -8,11 +8,19 @@ import (
 // Permission stores the name of a permission along whether it is database scoped. This is used to later obtain a
 // formatted permission string for checking.
 type Permission struct {
-	PermissionName string
-	DatabaseScoped bool
+	PermissionName     string
+	DatabaseScoped     bool
+	IsCollectionFormat bool
 }
 
 func (perm *Permission) FormattedName(bucketName string) string {
+	if perm.IsCollectionFormat {
+		if perm.DatabaseScoped {
+			return fmt.Sprintf("cluster.collection[%s:_default:_default]%s", bucketName, perm.PermissionName)
+		}
+		return fmt.Sprintf("cluster.collection[*:*:*]%s", perm.PermissionName)
+	}
+
```

Comment:
Do we have (or anticipate) any permissions where DatabaseScoped != IsCollectionFormat?  I don't see any today - am curious which permissions would be using the `cluster.bucket[%s]%s` format.

Reply:
A few existing permissions on CBS still use that older format but true, there are none that we use that are in that format now so we can probably remove that. 
Similarly the case of `cluster.collection[*:*:*]%s` is now no longer needed as Bryan changed the format of the dev_ops permission in the latest fix. Will remove those things and probably strip down just to database scoped.